spec/models/victim_liaison_officer_spec.rb

```diff
@@ -7,20 +7,20 @@
     expect(subject).to validate_presence_of :email
   end
 
-  context 'with case information record' do
+  context 'with an offender record' do
     before do
-      create(:case_information, victim_liaison_officers: [build(:victim_liaison_officer)])
+      create(:offender, victim_liaison_officers: [build(:victim_liaison_officer)])
     end
 
-    let(:ci) { CaseInformation.last }
+    let(:ci) { Offender.last }
```

Comment:
This variable could probably have a better name now.

```suggestion
    let(:offender) { Offender.last }
```